### PR TITLE
Only chrome latest

### PR DESCRIPTION
**Pull request by calebjclark**
All other emulators have moved to their own repos.

emulate-chrome-88 is a submodule in order for chrome-latest to pull in during development mode.

### CODE REVIEW

File: core/test/user-profile.test.ts [hunk 1]
```diff
@@ -133,49 +132,6 @@ describe('UserProfile cookie tests', () => {
       await tab.close();
     }
   });
-
-  it('restores cookies for safari', async () => {
```
Comment (blakebyrnes) [inline core/test/user-profile.test.ts:137]: Do you think we'll just throw out this test? Or would it move into the safari repo?

File: yarn.lock [hunk 2]
```diff
@@ -2684,6 +2684,31 @@
...
 
+"@secret-agent/core-interfaces@1.4.1-alpha.4":
+  version "1.4.1-alpha.4"
+  resolved "https://registry.yarnpkg.com/@secret-agent/core-interfaces/-/core-interfaces-1.4.1-alpha.4.tgz#912792be401298b747981a75786c86d7622af393"
+  integrity sha512-cge9C5NhLEVU5b8aBniPBGwwTjtqSDOVS+vi+GeNmQlsfcC97s9bMxXymBWi/WU/5rAY7Tn4UfsoJLvQfqIwgg==
+  dependencies:
+    "@secret-agent/puppet-interfaces" "1.4.1-alpha.4"
+    awaited-dom "^1.1.12"
+
+"@secret-agent/emulate-safari-13@1.4.1-alpha.4":
```
Comment (blakebyrnes) [inline yarn.lock:2695]: Should this dependency be here?

File: yarn.lock [hunk 3]
```diff
@@ -2684,6 +2684,31 @@
   dependencies:
     "@octokit/openapi-types" "^5.3.2"
 
+"@secret-agent/core-interfaces@1.4.1-alpha.4":
```
Comment (blakebyrnes) [inline yarn.lock:2687]: Didn't this get removed?

**Review by blakebyrnes — COMMENTED** [review]
Do you have a sense of how we'll manage dependencies for the new emulator packages? I assume lerna won't work across the repos? Right now, it looks like the emulators depend on puppet, interfaces and commons (for logger and createPromise). If we introduce a new version of puppet, any sense of how we can maintain compatibility? Will the new repos watch the main branch and test against them or something?

**Comment by calebjclark** [comment]
I haven't fully figured that out.... my current thought is we test emulators against the main SA repo using yarn workspaces from the emulator repos.

File: yarn.lock (hunk 3 above)
Comment (calebjclark) [reply yarn.lock]: Removed

File: yarn.lock (hunk 2 above)
Comment (calebjclark) [reply yarn.lock]: Removed

File: core/test/user-profile.test.ts (hunk 1 above)
Comment (calebjclark) [reply core/test/user-profile.test.ts:137]: I moved it into the safari repo.

**Pull request merged**